lib/maxmind-db-downloader.js

```diff
@@ -95,7 +95,9 @@ var MaxmindDbDownloader = function () {
               }, function (err) {
                 // download resulted in an error, do not rename
                 // remove temp file
-                fs.unlinkSync(targetFilePathTemp);
+                if (fs.existsSync(targetFilePathTemp)) {
```

Comment:
Let's wait to see what happens in node land. I'm guessing this will be a larger problem with node 8